onnx/backend/test/cpp/driver_test.cc

```diff
@@ -322,6 +322,17 @@ class ONNXCppDriverTest : public ::testing::TestWithParam<
                 true);
           }
         }
+        /*
+         * Examine functions in onnxifi_ext
+         */
+#ifdef ONNXIFI_ENABLE_EXT
```

Comment:
ENABLE_ONNXIFI_EXT might be more natural?

Reply:
Well, since all onnxifi related macros are start with an "ONNXIFI_...", I wonder if we really want to change this.